drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties

```diff
@@ -36,6 +36,7 @@ ValueList=Value list
 ValueListsExplanation=Value lists are an optional comma separated list of values to show as a drop down.
 optionalValueList=(optional) value list:
 DefaultValue=Default value
+DefaultValueExplanation=Change of ''Default value'' will affect just the new table rows.
```

Comment:
Are you sure that there should be a period? I see some notifications are with it and without that makes it strange.
And the neighbor notification got no period as well WDYT?
![Period](https://user-images.githubusercontent.com/21653197/82995759-4bc90500-a004-11ea-93bd-0135af657953.png)

Reply:
@dupliaka thank you, applied in a separate commit.